homeassistant/components/media_player/plex.py

```diff
@@ -457,6 +479,15 @@ def force_idle(self):
         self._session = None
         self._clear_media_details()
 
+    @asyncio.coroutine
```

Comment:
we're using async/await now.

So remove decorator and make it `async def async_added_to_hass`.